lecture_1/homework/app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,20 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  included do
+    rescue_from ActiveRecord::RecordInvalid, with: :four_twenty_two
```

Comment:
Ładniej by to jeszcze wyglądało gdybyś skorzystał z bloku zamiast customowych metod :) np.:
```ruby
rescue_from ActiveRecord::RecordNotFound do |exception|
    render json: { message: exception.message }, status: 404
  end

  rescue_from ActiveRecord::RecordInvalid do |exception|
    render json: { message: exception.message }, status: 422
  end
```